platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -124,6 +125,7 @@ public class PlatformWiring implements Startable, Stoppable, Clearable {
     private final EventStreamManagerWiring eventStreamManagerWiring;
     private final RunningHashUpdaterWiring runningHashUpdaterWiring;
     private final IssDetectorWiring issDetectorWiring;
+    private final AppCommunicationComponentWiring appCommunicationComponentWiring;
```

Comment:
Now that the app communication component has its own wiring object, its soldering should be moved out of the `wireExternalComponents` method, and into `wire`

Reply:
updated.